### PR TITLE
[fix] not take into account deleted stop in check_stop_time_consistency

### PR DESCRIPTION
**Pull request by patochectp**
ref: ND-1213

### CODE REVIEW

**Review by woshilapin — APPROVED** [review]
Code looks fine and from what I understand, business logic is fine but probably someone else with more context is needed for the review (:eyes: @pbougue :smile:)

File: tests/integration/piv_test.py [hunk 1]
```diff
@@ -1980,3 +1980,48 @@ def test_piv_trip_creation_existing_company_and_empty_physical_mode(mock_rabbitm
...
+def test_piv_re_routed_at_the_beginning_with_hourly_decrease(mock_rabbitmq):
+    """
+    old trip
+       "85010231" - "85010157" - "85010140" - "85162735" - "85162750" - "87745497"
+
+    new trip
+                    "85010157" - "85010140" - "85162735" - "85162750" - "87745497"
+                      //
+    "85010116" - "85010082"
+    """
```
Comment (pbougue) [inline tests/integration/piv_test.py:1994]: Can you add a bit of explanation here?
Maybe hours, order of stops deleted, added (and 85010157) and a sentence like:
`Datetimes of stops are not correctly sorted when comparing to "rang", but it is correct anyway, as only deleted stops are incorrectly sorted.`

**Review by pbougue — CHANGES_REQUESTED** [review]
I think we should change code, but I'm not sure that it's worth adding tests for those "extreme" cases (needs a bit of thinking to find a case that breaks with your current implem and not with a more precise implem).

**Review by pbougue — APPROVED** [review]

File: kirin/piv/model_maker.py [hunk 2]
```diff
@@ -459,12 +466,24 @@ def _make_trip_update(self, json_train, vj):
...
+                    if arret.get(STOP_EVENT_MDI_MAP[event_toggle])
+                    else False
+                )
+                rt_stop_time[event_toggle] = (
+                    event_datetime
+                    if event_datetime and not ((event_status and event_status in DELETED_STATUSES) or mdi)
+                    else datetime.datetime.fromtimestamp(0)
+                )
+
             _check_stop_time_consistency(
```
[the remark this answers is not in the document]
Comment (patochectp) [reply kirin/piv/model_maker.py]: done in 5b964ccf72098af795b13082ff0077e59bdb0f5d

File: tests/integration/piv_test.py [hunk 3]
```diff
@@ -1980,3 +1994,86 @@ def test_piv_trip_creation_existing_company_and_empty_physical_mode(mock_rabbitm
...
+    assert "invalid feed: stop_point's(85010140) time is not consistent" in res.get("error")
+
+    # "85010140" should not be taken into account in _check_stop_time_consistency
+    ads = piv_feed["objects"][0]["object"]["listeArretsDesserte"]["arret"]
+    for desserte in ads:
+        if desserte["emplacement"]["code"] == "85010140":
+            desserte["indicateurDescenteInterdite"] = True
+            desserte["indicateurMonteeInterdite"] = True
+
+    api_post("/piv/{}".format(PIV_CONTRIBUTOR_ID), data=ujson.dumps(piv_feed))
```
[the remark this answers is not in the document]
Comment (patochectp) [reply tests/integration/piv_test.py]: done in 5b964ccf72098af795b13082ff0077e59bdb0f5d

**Review by pbougue — APPROVED** [review]

**Pull request merged**